kyuubi-server/src/main/scala/org/apache/kyuubi/server/JettyServer.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server
+
+import org.eclipse.jetty.server.{HttpConfiguration, HttpConnectionFactory, Server, ServerConnector}
+import org.eclipse.jetty.server.handler.{ContextHandlerCollection, ErrorHandler}
+import org.eclipse.jetty.servlet.ServletContextHandler
+import org.eclipse.jetty.util.component.LifeCycle
+import org.eclipse.jetty.util.thread.{QueuedThreadPool, ScheduledExecutorScheduler}
+
+import org.apache.kyuubi.Utils
+
+class JettyServer private (
+    server: Server,
+    connector: ServerConnector,
+    rootHandler: ContextHandlerCollection) {
+  def start(): Unit = {
+    server.start()
+    connector.start()
+    server.addConnector(connector)
+  }
+
+  def stop(): Unit = {
+    server.stop()
```

Comment:
Should the server stop after the connector stops?

Reply:
make sense, thanks